modules/c++/math.linear/include/math/linear/Matrix2D.h

```diff
@@ -1082,6 +1082,21 @@ class Matrix2D
     }
 
     /*!
+     *  Negation operator;
+     *
+     *  \code
+           B = -A;
+     *  \endcode
+     *
+     */
+    Matrix2D operator-() const
+    {
+        Matrix2D mneg(*this);
+        std::transform(mneg.mRaw, mneg.mRaw+mneg.mMN, mneg.mRaw, std::negate<_T>());
```

Comment:
Yeah that's the motivation... to always make sure the header is self-contained.  In real life, `<functional>` is probably usually getting pulled in elsewhere but this way we're safe